### DISCUSSION TITLE
A new report has been submitted through Druid API but let us know how to publish it on the portal.

### DISCUSSION THREAD

**Discussion by kamal-singhal** [Sunbird-Obsrv/Community#133] [Issues] [answered]
Hi Team,
A new report has been submitted through below Druid API.

https://diksha.gov.in/api/data/v1/report/jobs/submit

But don't know how to publish it on the portal. Please let us know the process how to publish the new report to portal through API. 
we are waiting your response, this is highest priority requirement.

**Comment by manjudr** [comment, marked as answer]
@kamal-singhal  - I hope this discussion is related to https://github.com/orgs/Sunbird-Obsrv/discussions/131. 

Please go through the below developer setup documentation and let us know if you are encountering any issues - 

Developer Guide - https://obsrv.sunbird.org/learn/product-and-developer-guide/report-configurator
